llvm/lib/CodeGen/StackColoring.cpp

```diff
@@ -561,8 +580,15 @@ LLVM_DUMP_METHOD void StackColoring::dump() const {
 
 LLVM_DUMP_METHOD void StackColoring::dumpIntervals() const {
   for (unsigned I = 0, E = Intervals.size(); I != E; ++I) {
-    dbgs() << "Interval[" << I << "]:\n";
+    dbgs() << "Interval[" << I << "]:";
+    if (MFI->getObjectAllocation(I))
+      dbgs() << *MFI->getObjectAllocation(I);
+    dbgs() << "\n";
     Intervals[I]->dump();
+    dbgs() << "LiveStarts:";
```

Comment:
```suggestion
    dbgs() << '\n' << *Intervals[I];
    dbgs() << "LiveStarts:";
```

Not sure without checking if LiveIntervals's printer includes the newline or not